packages/gamut-labs/src/brand/Testimonial/index.tsx

```diff
@@ -47,7 +47,6 @@ export const Testimonial: React.FC<TestimonialProps> = ({
             <div className={s.avatarContainer}>
               <Avatar
                 src={imageUrl}
```

Comment:
Can we also just supply an empty `alt` tag?

[TestIO Comment:](https://codecademy.test.io/products/9763/test_cycles/63863/bugs/1333780)
>Actual Result
The alternative of the images duplicates adjacent text, thus making it announce twice, e.g. 'Photo of Lacey Bathala, Lacey Bathala ...'. **This makes the image decorative and it should have an empty alt tag** - fails 1.1.1
```suggestion
                src={imageUrl}
                alt=''
```

Reply:
Oh, that's totally right 🤦 oops